preferences.json

```diff
@@ -0,0 +1,13 @@
+{
+  "server": {
+    "thumbnailMaxWidth": 300,
+    "thumbnailMaxHeight": 300,
+    "hoursSchedule": [
+      0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23
+    ]
+  },
+  "web": {
+    "paginationPresets": [2, 5, 10, 20, 50],
+    "defaultLanguageCode": "en_nz"
```

Comment:
Ok, the default is defined here... I think this answers one question I had while reviewing the Front-end PR. 😄